src/index.ejs

```diff
@@ -19,6 +19,17 @@
   <script src="https://cdn.jsdelivr.net/npm/import-map-overrides/dist/import-map-overrides.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/system.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/amd.js"></script>
+  <template id="single-spa-layout">
+    <single-spa-router>
+        <application name="@vue-mf/navbar" loader="navbar" props="githubLink"></application>
+        <route path="view-doggos">
```

Comment:
Let's put the two `<route>` elements under a `<main>` element, since the view and rate MFEs are main content